docs/course_materials/compound_data_types.html

```diff
@@ -50,130 +54,182 @@ <h3><b>Basics</b></h3>
                     </code></pre>
                     <ul>
-                        <li>The .extend() method adds all the elements of an iterable to the end of the list:</li>
+                        <li>Delete an item by index:</li>
                     </ul>
                     <pre><code data-trim data-noescape class="hljs lang-python">
-lst = [1,2,3]
-lst2 = [4,5,6]
-lst.extend(lst2)
-print(lst)      # [1, 2, 3, 4, 5, 6]
-print(len(lst)) # 6
+                        del my_list[0]
+                        print(my_list) # ['apple', 'plum', 'melon', 'banana']
                     </code></pre>
-                </section>
-                <section>
-                    <h3><b>Retrieving elements by index, negative index, and slicing:</b></h3>
+                    <ul>
+                        <li>Deleting an item by value:</li>
+                    </ul>
                     <pre><code data-trim data-noescape class="hljs lang-python">
-print(l[0])     # 'plum'
-print(l[8])     # IndexError: list index out of range
-print(l[-1])    # 'melon'
-print(l[0:2])   # ['plum', 'melon']
+                        my_list.remove('plum')
+                        print(my_list) # ['apple', 'melon', 'banana']
                     </code></pre>
                 </section>
                 <section>
-                    <h3><b>Concatenation and multiplication:</b></h3>
+                    <h3><b>Deleting items from a list #2</b></h3>
+                    <h4>Another handy method is .pop()</h4>
                     <ul>
-                        <li>Lists can be concatenated with + and multiplied by *:</li>
+                        <li>Removes an element from the list</li>
+                        <li>Returns the value of the removed element</li>
                     </ul>
                     <pre><code data-trim data-noescape class="hljs lang-python">
-l = [25, 3, 14] + [5, 4] + [101, 2]
-print(l)    # [25, 3, 14, 5, 4, 101, 2]
-
-l = ["foo", "bar"] * 3
-print(l)    # ['foo', 'bar', 'foo', 'bar', 'foo', 'bar']
+                        my_list = ['orange', 'apple', 'plum', 'melon', 'banana']
+                        banana = my_list.pop()
+                        print(banana) # 'banana'
+                        print(my_list) # ['orange', 'apple', 'plum', 'melon']
+                    </code></pre>
+                    <ul>
+                        <li>By default, removes the element from the end of the list</li>
+                        <li>Index can be specified for an item to be popped</li>
+                    </ul>
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+                        my_list.pop(0) # 'orange'
+                        print(my_list) # ['apple', 'plum', 'melon']
                     </code></pre>
                 </section>
                 <section>
-                    <h3><b>In-place list operations</b></h3>
+                    <h3><b>Joining lists</b></h3>
```

Comment:
I would split this into two slides : 
- extend a list
- joining to list

Reply:
Sorry, can you give an example?

Reply:
@EgorLu sure. 
I just think 'joining lists'  more suitable to  [] + [] 
and 'extend a list' is more suitable to extend()